### DISCUSSION TITLE
加速偶尔不成功

### DISCUSSION THREAD

**Discussion by ykh123456** [BeyondDimension/SteamTools#2371] [Q&A]
有时候使用加速显示加速成功没有效果，断网重连或者重启电脑再使用加速才有效果，请问是为什么

**Comment by elarbor** [comment]
我也是 vercel.app 的不成功
![image](https://github.com/user-attachments/assets/e81f2144-d9a8-474e-ab28-8f2c5f2a8339)
![image](https://github.com/user-attachments/assets/022177de-237a-4071-9174-13d1cd50ac9d)
![image](https://github.com/user-attachments/assets/7e3d097b-4b57-4858-a691-9f429402cfac)
